src/HybridDb/Migrations/BuiltIn/RecreateMessageTablesAfterSchemaChanges.cs

```diff
@@ -32,14 +28,15 @@ public override void Execute(DocumentStore store)
                 var columns = string.Join(", ", columnNames);
 
                 // Move the data from the old to the new table
+                // For now we must handle the not nullable field CorrelationId manually.
                 store.Database.RawExecute(@$"
-                    insert into {tableNameEscaped} ({columns})
-                    select {columns} from {oldTableNameEscaped};");
+                    insert into {tableNameEscaped} ({columns}, CorrelationId)
+                    select {columns}, 'N/A' from {oldTableNameEscaped};");

```

Comment:
Var det ikke nok med en default value?

Reply:
Jeg tænker det kan være problematisk at gøre sådan her, for så skal vi til at lave versioner af denne migrering, for vi ved ikke hvornår den bliver kørt. Hvis vi kan komme på noget, hvor det ikke er nødvendigt, så lad os gøre det....

Reply:
Det var et levn fra tidligere - fjernet.